js/apps/admin-ui/src/identity-providers/ManageOrderDialog.tsx

```diff
@@ -57,11 +54,24 @@ export const ManageOrderDialog = ({
     setOrder(providerOrder);
   };
 
+  useFetch(
+    async () => adminClient.identityProviders.find(),
```

Comment:
Should we be passing in a `max` here as well? Like the call in IdentityProvidersSection?

Reply:
This list can be quite long for some users right? Should we not instead be doing this sorting in the REST API itself?